go/src/lib.rs

```diff
@@ -1,40 +1,73 @@
 /*
- * Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0
- */
-
-// TODO: Investigate using uniffi bindings for Go instead of cbindgen
```

Comment:
Since everybody aligned on the performance stats generated by using cbindgen, considering to go forward with cbindgen for now. Please let me know if we should spend time on exploring uniffi and if it can help improve our case